### ISSUE TITLE
heatmap() function would be useful before running model

### ISSUE THREAD

**Issue by jellevanzweden** [markvanderloo/rtrim#21]
Hi, 

in the related issue about the check_observations() function (https://github.com/markvanderloo/rtrim/issues/20), I came across errors that are presumably due to not having enough data in some years or using too short a period for the analysis. It would thus be very useful to be able to explore the data more thoroughly before running an rtrim model, for example with the heatmap() function. However, the heatmap() function requires the output of a call to trim, which would make it impossible to explore your data this way if the model run gives you an error. Catch 22.

Is it possible to adjust the heatmap() function so as to make it possible to view raw data? Or is it possible to build in other useful exploratory functions?

**Comment by pwbogaart** [comment]
Agreed. Maybe "model=0" (just the mean of all observed counts) would be a good mechanism to implement this.
